e2e/angular-core/src/ng-add.test.ts

```diff
@@ -72,7 +72,7 @@ describe('convert Angular CLI workspace to an Nx workspace', () => {
   });
 
   afterEach(() => {
-    cleanupProject();
+    cleanupProject({ skipReset: true });
```

Comment:
These tests do not start with standard create-nx-workspace so running `nx reset` fails after the test.

Reply:
They don't start with `create-nx-workspace` but still end up being Nx workspaces at the end of the tests. Each test converts the initial Angular CLI workspace into an Nx workspace. Surely that starts the daemon and needs to be reset, isn't it?

I can see a couple of test cases don't run any command, so maybe those don't have the daemon running, but the rest run commands. We might need to inline the `cleanupProject` calls to the end of each test case and pass the `skipReset: true` to those than don't run commands.

Reply:
Actually, by ensuring proper cleanup the `nx reset` issue is now gone so I reverted in back to clean `cleanupProject`